gourmet/keymanager.py

```diff
@@ -142,16 +142,20 @@ def look_for_key (self, txt):
         The higher the score, the more probable the match.
         """
 
-        txt = txt.lower()
+        txt = txt.casefold()
         retvals = {}
```

Comment:
I know this is not what you looked at, but do we want to make this a `defaultdict(float)`? Then we would remove the lines down below that set `0` for every value not in `retvals`. Might be more readable for the next guy and that might be you :D

Reply:
Makes sense!